### PR TITLE
Added pending queue

### PR DESCRIPTION
**Pull request by jbolles94**

### CODE REVIEW

File: src/packingSlip/controller.js [hunk 1]
```diff
@@ -198,6 +200,21 @@ async function GetPopulatedPackingSlips(
       }
     }
 
+    if (getPending) {
+      pipeline.push({
+        $match: {
+          $or: [
+            {
+              shipment: { $eq: null },
```
Comment (joshplumley) [inline src/packingSlip/controller.js:208]: Should we check for undefined as well?

File: src/packingSlip/controller.js [hunk 2]
```diff
@@ -385,6 +402,37 @@ async function getAllPackingSlips(_req, res) {
...
+        undefined,
+        false,
+        undefined,
+        undefined,
+        0,
+        undefined,
+        true,
+        true
+      );
+      if (e) return HTTPError("Error fetching packing slip history.");
```
Comment (joshplumley) [inline src/packingSlip/controller.js:423]: Should we mention pending here

File: src/packingSlip/controller.js [hunk 3]
```diff
@@ -385,6 +402,37 @@ async function getAllPackingSlips(_req, res) {
...
+
+      return {
+        data: {
+          packingSlips,
+        },
+      };
+    },
+    res,
+    "fetching packing slips"
```
Comment (joshplumley) [inline src/packingSlip/controller.js:432]: pending?

File: src/packingSlip/controller.js (hunk 1 above)
Comment (jbolles94) [reply src/packingSlip/controller.js:208]: As far as I am aware, this check should suffice in MongoDB wortld.

File: src/packingSlip/controller.js (hunk 3 above)
Comment (jbolles94) [reply src/packingSlip/controller.js]: Done.

File: src/packingSlip/controller.js (hunk 2 above)
Comment (jbolles94) [reply src/packingSlip/controller.js]: Done.

**Review by joshplumley — APPROVED** [review]

**Pull request merged**